tools/Tools.Common/Tools.Common.Netcore.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.0.6" />
     <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
+    <PackageReference Include="System.DirectoryServices.Protocols" Version="5.0.1" />
```

Comment:
Like `System.Security.Cryptography` and `System.Data.SqlClient`, we are update these assembly because the older versions have vulnerabilities.
Those assemblies are indirect references of Azure PowerShell, so we pin a fixed version here to make sure we are safe.